src/features/link/components/links-menu.tsx

```diff
@@ -0,0 +1,155 @@
+  DropdownMenuTrigger,
+} from "@/features/list";
+import { LinkCategory } from "@/types/enums";
+
+type Link = {
+  url: string;
+  category: LinkCategory;
+};
+
+const LINKS_DATA: Link[] = [
+  {
+    url: "https://www.youtube.com/",
+    category: "youtube",
+  },
+  {
+    url: "https://www.instagram.com/",
+    category: "instagram",
+  },
+  {
+    url: "https://www.tiktok.com/",
+    category: "tiktok",
+  },
+  {
+    url: "https://twitter.com/home",
+    category: "twitter",
+  },
+  {
+    url: "https://www.facebook.com/",
+    category: "facebook",
+  },
+  {
+    url: "https://www.google.com/",
+    category: "hp",
+  },
+  {
+    url: "https://www.google.com/",
+    category: "other",
+  },
+];
+
+const orderedCategories: LinkCategory[] = ["youtube", "instagram", "tiktok", "twitter", "facebook"];
+
+export const LinksMenu = () => {
+  const socialMediaLinks = LINKS_DATA.filter((data) => data.category !== "other" && data.category !== "hp");
+
+  const sortedSocialMediaLinks = socialMediaLinks.sort((a, b) => {
+    return orderedCategories.indexOf(a.category) - orderedCategories.indexOf(b.category);
+  });
+  const outsideIconLinks = sortedSocialMediaLinks.splice(0, 2);
+
+  const otherLinks = LINKS_DATA.filter((data) => data.category === "other" || data.category === "hp");
+
+  return (
+    <div className={"flex gap-x-4"}>
+      {outsideIconLinks.length > 0 &&
+        outsideIconLinks.map((outsideIconLink, index) => (
+          <LinksMenuOutsideIcon key={`links-menu-outside-icon-${index}`} link={outsideIconLink} />
+        ))}
+
+      <DropdownMenu>
+        <DropdownMenuTrigger className={"-mb-0.5 -mr-0.5"}>
+          <TbDotsCircleHorizontal className={"text-mauve-12 hover:text-mauve-11"} size={24} />
+        </DropdownMenuTrigger>
+
+        <DropdownMenuContent className={"p-0"}>
+          {sortedSocialMediaLinks.length > 0 &&
+            sortedSocialMediaLinks.map((socialMediaLink, index) => (
+              <LinksMenuItem key={`social-media-link-${index}`} link={socialMediaLink} />
+            ))}
+          {socialMediaLinks.length > 0 && otherLinks.length > 0 && <DropdownMenuSeparator className={"m-0"} />}
+          {otherLinks.length > 0 &&
+            otherLinks.map((otherLink, index) => <LinksMenuItem key={`other-link-${index}`} link={otherLink} />)}
+        </DropdownMenuContent>
+      </DropdownMenu>
+    </div>
+  );
+};
+
+const LinksMenuOutsideIcon = ({ link }: { link: Link }) => {
+  const icon = match(link)
+    .with({ category: "youtube" }, () => <TbBrandYoutube size={24} />)
+    .with({ category: "instagram" }, () => <TbBrandInstagram size={24} />)
+    .with({ category: "tiktok" }, () => <TbBrandTiktok size={24} />)
+    .with({ category: "twitter" }, () => <TbBrandTwitter size={24} />)
+    .with({ category: "facebook" }, () => <TbBrandFacebook size={24} />)
+    .with({ category: "hp" }, () => <TbLink size={16} />)
+    .with({ category: "other" }, () => <TbLink size={16} />)
```

Comment:
@kotaro1574
レビュー遅くなってすみません。
ミスタイプかと思いますが、今後のotherのLinkが全面に出るレシピが発生する場合も想定して、sizeは24にしておいた方がよろしいのではと思いました。

```suggestion
    .with({ category: "hp" }, () => <TbLink size={24} />)
    .with({ category: "other" }, () => <TbLink size={24} />)
```

Reply:
それ以外は問題ありませんでしたので、上記修正対応後にapproveしますね。